cmd/kpod/logout.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/containers/image/pkg/docker/config"
+	"github.com/kubernetes-incubator/cri-o/libpod/common"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+var (
+	logoutFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "override-path",
+			Usage: "Override the default path of the authentication file",
+		},
+	}
+	logoutDescription = "Logout from a specific registry server"
```

Comment:
Remove the cached username/password for the registry server.

Reply:
fixed